Instructions for adding an api key (#24)
reshufflehq/map-markers: README.md

It displays a map on the screen and attempts to set the map's center to the user's browser location.
It also lets the user click the map to add markers. These markers are persisted.

## Configuring a Google API Key
This template does not include a Google API key. Without a key, Google's map display is  darkened, and the map is intended for development purposes only.

> The following steps detail how to include your own Google API key into the code:

Reviewer: ```suggestion
To get and include your own Google API key:
```

1. Obtain a key following [these instructions](https://developers.google.com/maps/documentation/embed/get-api-key) (Skip this step if you already have a key)

> 2. Create a .env file in the root directory of the template

Reviewer: Is it still a "template"?
